numpy/core/src/umath/loops_hyperbolic.dispatch.c.src

```diff
@@ -74,9 +74,131 @@
  *
  */
 #if NPY_SIMD_F64
+
+    // For architectures without efficient gather / scatter instructions, it is
+    // better to use a transposed LUT where we can load all coefficients for an
+    // index linearly.  In order to keep the same vertical calculation, we
+    // transpose the coef. into lanes.  2 lane transpose is all that's
+    // implemented so we require `npyv_nlanes_f64` == 2.
+    #if npyv_nlanes_f64 == 2
+    #define TANH_TRANSPOSED_LUT
+    // Define missing universal intrinsics used below
+    #if !defined(npyv_get_lane_u64)
+        #if defined(NPY_HAVE_ASIMD)
+            #define UNDEF_npyv_get_lane_u64
+            #define npyv_get_lane_u64 vgetq_lane_u64
+        #elif defined(NPY_HAVE_SSE41)
+            #define UNDEF_npyv_get_lane_u64
+            #define npyv_get_lane_u64 _mm_extract_epi64
```

Comment:
I don't see any actual use of `npyv_get_lane_u64` in the code.

Reply:
Removed thanks!